simulator/internal/internal_cross.go

```diff
@@ -0,0 +1,20 @@
+// +build !cgo
+
+// Package internal provides stubs when not using CGO.
+package internal
+
+import (
+	"errors"
+	"io"
+)
+
+// SetSeeds
+func SetSeeds(r io.Reader) {}
+
+// Reset
+func Reset(forceManufacture bool) {}
+
+// RunCommand passes cmd to the simulator and returns the simulator's response.
```

Comment:
Since the function always returns an error, the comments should just say this is a place holder function

Reply:
Added some comments